docs: resolves markdown error and misspells in "pushing and pulling artifacts" section (#230)
oras-project/oras-www: docs/how_to_guides/pushing_and_pulling.mdx

# Pushing

## Pushing artifacts with single file

> Pushing single file artifacts involves referencing the unique artifact type and a file.

Author: but the section is specifically for pushing artifacts with single file, so we would only need one file, not more than one, hence `at least one file` doesn't make sense to me. Please consider it and let me know, final call is obviously yours.

- A single file (`artifact.txt`) that represents overview content that might be displayed as a repository overview
- A collection of files (`docs/*`) that represents detailed content. When specifying a directory, ORAS will automatically tar the contents.

> See [OCI Artifacts](https://github.com/opencontainers/artifacts/blob/main/artifact-authors.md#defining-a-unique-artifact-type) for more details.

Reviewer: ```suggestion
See [OCI Artifacts](https://github.com/opencontainers/image-spec/blob/main/manifest.md#guidelines-for-artifact-usage) for more details.
```

Reviewer: https://github.com/opencontainers/artifacts is archived so we need to point it to a new repository.

## Using cache when pulling artifacts

> In order to save unnecessary network bandwidth and disk I/O, oras provides a solution to pull the artifacts into a local content-address storage (CAS) if the content does not exist, and then copy the artifact to the desired storage.
> 

Reviewer 2: ```suggestion
Oras pulls the artifact into a content-addressable storage (CAS) cache if the content is not available locally to save bandwidth and disk I/O.
Once the content is availabe in the cache, oras copies the artifact to the desired location. 
```

@Wwwsylvia does the able statement sound accurage. My hope is to split up the long sentence and call out that the cache is a CAS.

Author: thank you @sajayantony for helping

Reviewer 3: LGTM

## Using cache when pulling artifacts

> Oras pulls the artifact into a content-addressable storage (CAS) cache if the content is not available locally to save bandwidth and disk I/O.

Reviewer 4: ```suggestion
ORAS pulls the artifact into a content-addressable storage (CAS) cache if the content is not available locally to save bandwidth and disk I/O.
```

Capitalize the word "ORAS" everywhere for consistency.

> Once the content is availabe in the cache, oras copies the artifact to the desired location. 

Reviewer 4: ```suggestion
Once the content is availabe in the cache, ORAS copies the artifact to the desired location. 
```
